srcpkgs/foot/template

```diff
@@ -1,21 +1,24 @@
 # Template file for 'foot'
 pkgname=foot
-version=1.7.2
+version=1.8.0
 revision=1
 wrksrc=$pkgname
 build_style=meson
 configure_args="-Dwerror=false"
 hostmakedepends="pkg-config scdoc wayland-devel"
-makedepends="fcft-devel libxkbcommon-devel ncurses-devel tllist wayland-devel
- wayland-protocols"
+makedepends="fcft-devel libutf8proc-devel libxkbcommon-devel ncurses-devel tllist
+ wayland-devel wayland-protocols"
 depends="${pkgname}-terminfo>=${version}_${revision}"
 short_desc="Fast, lightweight and minimalistic Wayland terminal emulator"
 maintainer="Isaac Freund <ifreund@ifreund.xyz>"
 license="MIT"
 homepage="https://codeberg.org/dnkl/foot"
 changelog="https://codeberg.org/dnkl/foot/src/branch/master/CHANGELOG.md"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=0c5fa72a315b65100dfb8b7343212535e75e990b2a9500e028c6078c7915eb8a
+checksum=9269acfbef40cb284590de3bf70fc67997ffb6361dec4b4dddae7cdbf4ad6490
+
+# FIXME: Not being set automatically
+CFLAGS+="-I$XBPS_CROSS_BASE/usr/include/libutf8proc"
```

Comment:
You can (and should) just set it to `CFLAGS`. And when using `+=` with such variables, remember to add the space: `+=" -I..."`

Reply:
The pkg config file has: `includedir=${prefix}/include`, which is either wrong or incompatible with what `foot` expects.

https://github.com/JuliaLang/julia/blob/d7d2b0c692eb6ad409d7193ba8d9d42972cbf182/src/flisp/julia_extensions.c#L6 uses it like foot does, so that's the "correct" usage. And we pass a similar arg in the Julia template: `UTF8PROC_INC='${XBPS_CROSS_BASE}/usr/include/libutf8proc'`

I will try to patch the pc file and see if our other packages don't break (they almost certainly won't, since `-I/usr/include` is a compiler default and any package doing `#include "libutf8proc/utf8proc.h"` will just keep working). From what I can see, it's our package that's doing broken stuff, since we move the header into a directory but didn't change what the pkg-config file does (the file is in `srcpkgs/libut8proc/files` ;)

https://github.com/JuliaStrings/utf8proc looks even more attractive now, since it ships with its own pc file.

Regarding this specific PR, for accuracy the comment should be `FIXME: our pkg-config file for utf8proc sets the wrong include dir`